tasks/Module 4/task_5.py

```diff
@@ -0,0 +1,17 @@
+"""Find Mistakes in a Spreadsheet."""
+import ezsheets
```

Comment:
Please add this package to `requirements.txt`.

Reply:
added